pkg/codec/x264/bridge.h

```diff
@@ -0,0 +1,88 @@
+#include <errno.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <x264.h>
+
+typedef struct Slice {
+  unsigned char *data;
+  int data_len;
+} Slice;
+
+typedef struct Encoder {
+  x264_t *h;
+  x264_picture_t pic_in, pic_out;
+  x264_param_t param;
+} Encoder;
+
+Encoder *enc_new(x264_param_t param) {
+  Encoder *e = (Encoder *)malloc(sizeof(Encoder));
+
+  if (x264_param_default_preset(&e->param, "veryfast", "zerolatency") < 0)
+    goto fail;
+
+  /* Configure non-default params */
+  e->param.i_csp = param.i_csp;
+  e->param.i_width = param.i_width;
+  e->param.i_height = param.i_height;
+  e->param.i_fps_num = param.i_fps_num;
+  e->param.i_fps_den = 1;
+  // Intra refres:
+  e->param.i_keyint_max = param.i_fps_num;
```

Comment:
https://github.com/pion/mediadevices/blob/6fd00ab14568e16971363b1622ba58c9feb9b15b/pkg/prop/prop.go#L128
`Codec.KeyFrameInterval` can be used.

Reply:
Oops, makes sense. Updated it.